Sources/AuthFoundation/JWT/Internal/DefaultIDTokenValidator.swift

```diff
@@ -18,61 +18,72 @@ import CommonCrypto
 
 struct DefaultIDTokenValidator: IDTokenValidator {
     var issuedAtGraceInterval: TimeInterval = 300
+    var checks: [ValidationCheck] = ValidationCheck.allCases
+    
+    enum ValidationCheck: CaseIterable {
+        case issuer, audience, scheme, algorithm, expirationTime, issuedAtTime, nonce, authTime, maxAge, subject
+    }
     
     func validate(token: JWT, issuer: URL, clientId: String, context: IDTokenValidatorContext?) throws {
```

Comment:
You might want to change the `.swiftlint.yaml` to increase the cyclomatic_complexity threshold.  Lets set it to 25 or so.  I'm not a fan of that particular lint check, to be honest...

Reply:
Good point, I've updated this.  I did set a warning and error configuration.